docs/reference/components/sinks/redis.cue

```diff
@@ -0,0 +1,145 @@
+package metadata
+
+components: sinks: redis: {
+	title: "Redis"
+	classes: {
+		commonly_used: false
+		delivery:      "best_effort"
+		development:   "beta"
+		egress_method: "batch"
+		service_providers: []
+		stateful: false
+	}
+	features: {
+		buffer: enabled:      true
+		healthcheck: enabled: true
+		send: {
+			compression: enabled: false
+			batch: {
+				enabled:      true
+				common:       true
+				max_bytes:    null
+				max_events:   1
+				timeout_secs: 1
+			}
+			encoding: {
+				enabled: true
+				codec: {
+					enabled: true
+					default: null
+					enum: ["json", "text"]
+				}
+			}
+			request: enabled: false
```

Comment:
It seems like we do support the `request` parameters looking at the implementation. I think this should look something like https://github.com/timberio/vector/blob/63afa786c247a0ae899aeacb75134f19b6b81b33/docs/reference/components/sinks/aws_cloudwatch_logs.cue#L40-L50

But with different values. For example, I see the `timeout` configured below defaults to `1`.